compiler/damlc/tests/daml-test-files/Unserializable.daml

```diff
@@ -1,11 +1,13 @@
 -- Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its affiliates.
 -- All rights reserved.
 
--- @ERROR expected serializable type
```

Comment:
We want to test for this error message. Thus, we need to use the desugared form rather than the `template` syntax for this test.

Reply:
Done